environments/puppet/data/hosts/puppet.your.domain.yaml

```diff
@@ -61,5 +51,32 @@ rsyslog::udp_listen_address: '127.0.0.1'
 simp::classes:
   - 'simp::server'
 
-simp::server::classes:
-  - 'simp::puppetdb'
+### PuppetDB ###
+#
+# PuppetDB will not be enabled on your server unless you uncomment the
+# 'simp::server::classes' section below.
+
+# simp::server::classes:
+#  - 'simp::puppetdb'
+
+# The following items exist to properly configure PuppetDB if you choose to
+# enable it above.
+
+# The service name of the Puppet Server service for use with PuppetDB
```

Comment:
These should be commented out as well.  The user should be told to uncomment out the block of settings.  Yes I know having users uncomment out more than a few lines this is more error prone.  However, IMHO it does not make sense to have settings that are unused in this file.